src/XMakeBuildEngine/Definition/ToolsetReader.cs

```diff
@@ -205,42 +205,45 @@ ToolsetDefinitionLocations locations
                         // Other toolsets are installed in the xbuild directory
                         var xbuildToolsetsDir = Path.Combine(libraryPath, "xbuild");
```

Comment:
Nitpick - maybe we can add the `+ Path.DirectorySeparatorChar` here, instead of later. So, it's use with `Regex.Escape` later becomes easier to read.